pkgs/by-name/do/dosbox-staging/package.nix

```diff
@@ -154,6 +98,8 @@ stdenv.mkDerivation (finalAttrs: {
     popd
```

Comment:
```suggestion
    popd
    
    # Point desktop file to dosbox-staging
    substituteInPlace $out/share/applications/dosbox-staging.desktop \
    --replace-fail "Exec=dosbox" "Exec=dosbox-staging"
```

Reply:
Otherwise base DOSBox will run if the user has it in their path.

Reply:
https://github.com/NixOS/nixpkgs/blob/1a0c12d45e430c48b9c7038da373e265650ece6e/pkgs/by-name/do/dosbox-staging/package.nix#L93

the current postFixup creates a wrapper of `dosbox` which is under the hood `dosbox-staging`

Reply:
The current process does not support both installations and will conflict. `dosbox` binary `dosbox` wrapper have the same name. It's hard to determine which one will launch first. We can, however, not rename the dosbox-staging at all, and create a wrapper instead at .dosbox-staging-wrapped. @AndersonTorres thoughts, I am not particularly experienced with the wrapping process.

Reply:
I think we can discuss this independently, since it is more an enhancement than a bugfix.

@keenanweaver can you please open an issue ticket explaining your point, so we can track it?

Reply:
Just opened. https://github.com/NixOS/nixpkgs/issues/312101

Reply:
Resolving this, then.